pages/common/nxc-ldap.md

```diff
@@ -0,0 +1,24 @@
+# nxc ldap
+
+> Pentest and exploit Windows Active Directory Domains via LDAP.
+> More information: <https://www.netexec.wiki/ldap-protocol>.
+
+- Search for valid domain credentials by trying out every combination in the specified lists of [u]sernames and [p]asswords:
+
+`nxc ldap {{192.168.178.2}} -u {{path/to/usernames.txt}} -p {{path/to/passwords.txt}}`
+
+- Enumerate active domain users:
+
+`nxc ldap {{192.168.178.2}} -u {{username}} -p {{password}} --active-users`
+
+- Collect data about the targeted domain and automatically import these data into bloodhound:
```

Comment:
```suggestion
- Collect data about the targeted domain and automatically import these data into BloodHound:
```
Like in [docs](https://www.netexec.wiki/getting-started/bloodhound-integration).